Description: Use the multiarch libdir
Author: Victor Seva <linuxmaniac@torreviejawireless.org>

Use the multiarch libdir (rather than the lib64/ schema used by Upstream)

--- a/Makefile.defs
+++ b/Makefile.defs
@@ -411,15 +411,12 @@
 endif
 $(info target architecture <$(ARCH)>, host architecture <$(HOST_ARCH)>)
 
-LIBDIR ?=
+LIBDIR ?= 
 ifeq ($(LIBDIR),)
-ARCHBSZ= $(shell echo $(ARCH) | sed -e 's/.*64.*/64b/')
-ifeq ($(ARCHBSZ),64b)
-	LIBDIR = lib64
-else
-	LIBDIR = lib
-endif
+	ARCH_TRIPLET=$(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+	LIBDIR = lib/$(ARCH_TRIPLET)
 endif
+$(info LIBDIR <$(LIBDIR)>)
 
 # dirs
 cfg_dir = etc/$(MAIN_NAME)/
